donate/templates/pages/core/campaign_page_master.html

```diff
@@ -50,21 +50,6 @@ <h1 class="heading heading--primary hero__heading">{{ page.title }}</h1>
                         {{ page.intro|richtext }}
                     </div>
                 </div>
-                {% block logo_showcase %}
```

Comment:
Removing the `logo_showcase` block from all templates, per request of our stakeholders 👍